src/testkit/index.ts

```diff
@@ -11,3 +11,4 @@ export {HintDriver} from '../components/hint/Hint.driver';
 export {RadioButtonDriver} from '../components/radioButton/RadioButton.driver';
 export {RadioGroupDriver} from '../components/radioGroup/RadioGroup.driver';
 export {SortableListItemDriver} from '../components/sortableList/SortableListItem.driver';
+export {SliderDriver} from '../components/slider/slider.driver';
```

Comment:
better use the incubator slider driver. The old Slider will be replaced on the next major version

Reply:
There is no driver for the incubator slider, or did I miss it?
I might write it in the next phase and replace this

Reply:
Right, my bad